numpy/core/_ufunc_config.py

```diff
@@ -85,14 +85,11 @@ def seterr(all=None, divide=None, over=None, under=None, invalid=None):
 
     Examples
     --------
-    >>> old_settings = np.seterr(all='ignore')  #seterr to known value
-    >>> np.seterr(over='raise')
-    {'divide': 'ignore', 'over': 'ignore', 'under': 'ignore', 'invalid': 'ignore'}
-    >>> np.seterr(**old_settings)  # reset to default
-    {'divide': 'ignore', 'over': 'raise', 'under': 'ignore', 'invalid': 'ignore'}
-
+    >>> orig_settings = np.seterr(all='ignore')  #seterr to known value
     >>> np.int16(32000) * np.int16(3)
     30464
+    >>> np.seterr(over='raise')
+    {'divide': 'ignore', 'over': 'ignore', 'under': 'ignore', 'invalid': 'ignore'}
```

Comment:
It's weird that this doesn't change the value in the returned dict, but indeed that is the behavior (the change is only reflected after calling `geterr`.

Reply:
It is a normal pattern for getters: you get back the old value so that you can reset it.

Honestly, we should maybe just remove this API in a follow-up and force users to use the errstate as only real api...